e2e/cypress/integration/account_settings/security/access_history_spec.js

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @account_setting
+
+describe('Account Settings -> Security -> View Access History', () => {
+    before(() => {
+        // # Login as new user and visit town-square
+        cy.apiInitSetup({loginAfter: true}).then(({team}) => {
+            cy.visit(`/${team.name}/channels/town-square`);
+        });
+    });
+
+    beforeEach(() => {
+        // # Go to Account Settings
+        cy.toAccountSettingsModal();
+
+        // * Check that the Security tab is loaded
+        cy.get('#securityButton').should('be.visible');
+
+        // # Click the Security tab
+        cy.get('#securityButton').click();
+    });
+
+    it('MM-T2087 View Access History', () => {
+        // # Click "View Access History" link
+        cy.get('button[data-testid="viewAccessHistory"]').should('be.visible').click();
```

Comment:
nit: you may use `cy.findByTestId`
```suggestion
        cy.findByTestId('viewAccessHistory').should('be.visible').click();
```